packages/salesforcedx-vscode-soql/src/commonUtils.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { TextDocument } from 'vscode';
+
+export function getDocumentName(document: TextDocument): string {
+  const documentPath = document.uri.fsPath;
+  return documentPath.split('/').pop() || '';
```

Comment:
You can use `path.basename` to get the end part of the filepath.

Reply:
ooo cool, as in `document.uri.path.baseName`? Ha, if thats the case it seems silly to have a separate file, method and tests for something I can access directly =(

Reply:
got it, like this `path.basename(this.document.uri.fsPath)`

Reply:
Looks good to me, thanks.